buildscripts/testchangefeed.py

```diff
@@ -0,0 +1,448 @@
+#!/usr/bin/env python
+
+"""
+Run basic notary client tests against a server
```

Comment:
this should reflect `changefeed`.  Should this supersede the existing `testclient.py`?

Reply:
Long term yes but in the immediate future I'm not pushing on that.